docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -6,6 +6,16 @@ The MySQL connector allows querying and creating tables in an external
 `MySQL <https://www.mysql.com/>`_ instance. This can be used to join data between different
 systems like MySQL and Hive, or between two different MySQL instances.
 
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a MySQL data
+source are:
+
+* MySQL 5.6, 5.7, 8.0 and higher
+* Network access, by default on port 3306, from the Trino coordinator and
+  workers to MySQL.
```

Comment:
This sentence feels awkward to me. 

> "The default network access is port 3306 from the Trino coordinator and workers to MySQL." 
or
> "The default Trino coordinator and worker network access to MySQL is port 3306."

I'm leaning towards the 2nd sentence style. If there's a precedent I'm missing for the other sentence structure, let me know.